Sources/Extensions/NSAttributedStringExtension.swift

```diff
@@ -9,15 +9,15 @@
 import Cocoa
 
 public extension NSAttributedString {
-    public class func attributedStringFromHTML(html: String, withCSSString css: String?) -> NSAttributedString {
+    public class func attributedStringFromHTML(_ html: String, withCSSString css: String?) -> NSAttributedString {
```

Comment:
Shouldn't it be `open` instead of `public` ? 
Do we really want users to override it ?